frontend/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Login from './pages/Login.jsx';
+
+const App = () => {
+  return (
+    <div>
+      <Login classn />
```

Comment:
Did you misspell `className`?

Reply:
Oops! That’s a mistake, there should be no classn at all, thanks! Good eye!